package.json

```diff
@@ -91,7 +91,7 @@
     "fmt-check": "prettier --check ./src *.ts *.json *.js ./e2e ./packages ./rust/kcl-language-server",
     "fetch:wasm": "./scripts/get-latest-wasm-bundle.sh",
     "fetch:wasm:windows": "./scripts/get-latest-wasm-bundle.ps1",
-    "fetch:samples": "echo \"Fetching latest KCL samples...\" && curl -o public/kcl-samples-manifest-fallback.json https://raw.githubusercontent.com/KittyCAD/kcl-samples/next/manifest.json",
+    "fetch:samples": "rm -rf public/kcl-samples* && curl -L -o public/kcl-samples.zip https://github.com/KittyCAD/kcl-samples/archive/refs/heads/next.zip && unzip -o public/kcl-samples.zip -d public && mv public/kcl-samples-* public/kcl-samples",
```

Comment:
This is interesting: I understand this PR as making the modeling-app repo the source of truth for the kcl-samples, but this command seems to allow that to flow the other way?

Reply:
Ah no I don't think we should delete it, sorry I thought you added it to be used elsewhere in your changes and I just hadn't found how it was used yet.

Reply:
No, nothing uses this anymore. How about we leave it until the kcl-samples repo gets archived?